CONTRIBUTING.md

```diff
@@ -28,6 +28,11 @@ Alternatively, submit a pull request with one of the following
 * A proof-of-concept implementation.
 * In case the change is trivial, you can start with a draft or even provide a PR with the final implementation.
 
+When working on a pull request please make sure that you have followed [pull
```

Comment:
**Commit subject**:
s/meta/CONTRIBUTING

meta doesn't IMO hint at anything when it comes to narrowing the context of the changes in the subject.

Reply:
Done.